cmd/pack/logging_test.go

```diff
@@ -0,0 +1,60 @@
+package main
```

Comment:
We have a pattern in this codebase of not testing unexported types inside of the package under test (i.e., we always use `_test` packages).

Could this be an acceptance test (see the `acceptance` package)? Alternatively, could we move these tests and the associated code out of `cmd/pack` entirely?